### ISSUE TITLE
du -s * doesn't work

### ISSUE THREAD

**Issue by fdncred** [nushell/engine-q#835]
Trying to run the external `du` and not having much success.

```
> du -s *  
du: cannot access '*': No such file or directory
```

versus in fish

```
> du -s * 
96	Cargo.lock
4	Cargo.toml
3792	crates
44	docs
4	history.txt
4	LICENSE
4	README.md
152	src
5795680	target
4	TODO.md
```

**Comment by jntrnr** [comment]
I'm seeing the same thing in nushell:

```
> ^du -s * 
du: cannot access '*': No such file or directory
```

I think this is coming from us not auto-expanding the glob pattern like bash or fish will.

**Comment by kubouch** [comment]
Yeah, you can expand it with `(ls *).name`.

I'm wondering if we should have a `glob` command that would turn a glob pattern into a list. Or should we always auto-expand?

**Comment by fdncred** [comment]
I'm not sure what the right answer is either. I really just want things to 'just work' without having to remember 'tricks' to get them to work. Some things should just work, other things should work differently. It's hard to tell when to work for either way.

**Comment by jntrnr** [comment]
If it's an external argument, we could just auto-expand the glob like other shells do

**Comment by kubouch** [comment]
Yeah, that would be the most user-friendly solution. We should also have a way to disable the expansion (e.g., env var).

**Comment by jntrnr** [comment]
fixed by #839
